source/programme-talks.rst

```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
+
+About 57 of the speakers were male, and the vast majority were of course
+extremely white.
+
+Some of our best friends are white men, and there is nothing wrong with
+us^wthem, but all the same some of our best friends are not, and for some reason
+the ones who aren't didn't come forward to speak at PyCon.
+
+The simple solution to this is to get more of our friends who are not white men
+to propose talks.
+
+Target
+------
+
+I will be disappointed if at this PyCon we don't receive at least **one-third of
```

Comment:
It's not clear who "I" is here.  Since this document is going to be part of our conference policy, can you change "I" to "we", perhaps changing "I will" to "we should"?